src/components/navigation/breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../base/icons/Icon';
+
+function Breadcrumb({ breadcrumbClasses, children, ...props }) {
+  return (
+    <div className={breadcrumbClasses}>
+      {React.Children.map(children, (child, i) => (
+        <span key={name}>
+          {i > 0 && <Icon name="chevron-right" />}
+          {child}
+        </span>
+      ))}
+    </div>
+  );
+}
+
+Breadcrumb.propTypes = {
+  breadcrumbClasses: PropTypes.string,

```

Comment:
I tend to prefer the default classNames prop and just passing that through as opposed to having a special prop.  I've been doing that with some of the other components because it seems more intuitive than having to look up and find the right prop name to use for something there's already a name for.  Maybe there's arguments against doing that though.